examples/c/tcp-push-pop.c

```diff
@@ -95,18 +103,19 @@ static int push_get_sent_nbytes(const struct sockaddr_in *const addr, const int
     return sent_bytes;
 }
 
-static void run_client(const struct sockaddr_in *const addr)
+static void run_client(const struct sockaddr_in *const addr, size_t data_size, unsigned max_msgs)
 {
-    unsigned int sent_bytes = 0;
+    size_t sent_bytes = 0;
     int sockqd = connect_get_sockqd(addr);
+    size_t max_bytes = data_size * max_msgs;
 
-    while (sent_bytes < MAX_BYTES)
+    while (sent_bytes < max_bytes)
     {
-        sent_bytes += push_get_sent_nbytes(addr, sockqd);
-        fprintf(stdout, "push: total bytes sent: (%d)\n", sent_bytes);
+        sent_bytes += push_get_sent_nbytes(addr, sockqd, data_size);
+        fprintf(stdout, "push: total bytes sent: (%zu)\n", sent_bytes);
     }
 
-    assert(sent_bytes == MAX_BYTES);
+    assert(sent_bytes == max_bytes);
```

Comment:
Also fairly sure this assert is dead code.

Reply:
Indeed.